### PR TITLE
Add Kafka publishing for patches topic

### LINKED ISSUE #3: Implement Kafka publishing for patches
Publish patch messages to a pub/sub topic when a patch is made to a conversation so that other instances of `patches` can get the change and distribute it to the appropriate WebSockets and so that `ether` can update the stored copy of the content file.

### PR DESCRIPTION
**Pull request by igorveselinovic**
Resolves #3 

Added code that publishes EDIT Update type messages to the "updates" topic that `ether` will read from. For the purposes of just updating the content file, this should be enough since `ether` only needs the patches being processed by `patches` so that it can apply them.

Eventually, if/when we figure out how to deal with redundant copies of `patches`, we'd need to publish all Update messages (right now, just EDIT and CURSOR) to this topic so that the other copies of `patches` can send out all Update messages to their respective clients. But for now, this is just to get `ether` and `patches` hooked up.

Added a new `errc` channel for the `Conversation` struct, so asynchronous tasks like writing to Kafka (and eventually writing to TimescaleDB) write errors to this channel. Right now, if this channel receives an error, the whole conversation just shuts down.

### CODE REVIEW

File: kafka/writer.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	err = k.patchesWriter.WriteMessages(context.Background(),
+		segkafka.Message{
+			Key:   []byte(strconv.FormatInt(conversationID, 10)),
+			Value: pubBytes,
+		},
+	)
+	if err != nil {
+		return err
+	}
+	return nil
```
Comment (Thao-Tran) [inline kafka/writer.go]:
```suggestion
	return err
```
or you can just return when you call the function.

File: websockets/conversation.go [hunk 2]
```diff
@@ -41,6 +52,9 @@ func NewConversation(conversationID int64, doc string) *Conversation {
 		register:       make(chan *Client),
 		unregister:     make(chan *Client),
 		broadcast:      make(chan *BroadcastMessage),
+		errc:           make(chan error),
+		db:             db,
```
Comment (Thao-Tran) [inline websockets/conversation.go:56]: Add a `TODO` for where we'll use the `db` connection.

**Review by Thao-Tran — CHANGES_REQUESTED** [review]
Also make an issue for integrating DB stuff.

File: kafka/writer.go (hunk 1 above)
Comment (igorveselinovic) [reply kafka/writer.go]: I'm dumb

Comment (Thao-Tran) [reply kafka/writer.go]: It happens to the best of us

**Review by Thao-Tran — APPROVED** [review]

**Pull request merged**